alodataset/coco_detection_dataset.py

```diff
@@ -174,6 +185,9 @@ def getitem(self, idx):
             target["boxes"] = target["boxes"][idxs]
             target["labels"] = torch.from_numpy(new_labels[idxs])
 
+            if self.prepare.return_masks:
```

Comment:
Isn't that supposed to be in the coco_panoptic only ?

Reply:
Let me know what you think, Just asking

Reply:
@thibo73800, if annotations has "segmentation" attribute, any dataset could be represented as segmentation. For example, instance_train2017.json and instance_val2017.json coco annotations have this atribute. So, it is possible get the mask for this dataset.

Reply:
got it